clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10490,16 +10490,35 @@ static void getSPIRVBackendOpts(const llvm::opt::ArgList &TCArgs,
       TCArgs.MakeArgString("--avoid-spirv-capabilities=Shader"));
   BackendArgs.push_back(
       TCArgs.MakeArgString("--translator-compatibility-mode"));
-  // TODO: A list of SPIR-V extensions that are supported by the SPIR-V backend
-  // is growing. Let's postpone the decision on which extensions to enable until
-  // - the list is stable, and
-  // - we decide on a mapping of user requested extensions into backend's ones.
-  // Meanwhile we enable all the SPIR-V backend extensions.
-  BackendArgs.push_back(TCArgs.MakeArgString("--spirv-ext=all"));
-  // TODO:
-  // - handle -Xspirv-translator option to avoid "argument unused during
-  // compilation" error
-  // - handle --spirv-ext=+<extension> and --spirv-ext=-<extension> options
+
+  // Disable all the extensions by default
```

Comment:
Could you please update the comment to add why we are disabling all the SPIR-V backend extensions to give some additional context?